code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -0,0 +1,182 @@
+#define ETHEREAL_COLORS list("#00ffff", "#ffc0cb", "#9400D3", "#4B0082", "#0000FF", "#00FF00", "#FFFF00", "#FF7F00", "#FF0000")
+
+/datum/species/ethereal
+	name = "Ethereal"
+	id = "ethereal"
+	attack_verb = "burn"
+	attack_sound = 'sound/weapons/etherealhit.ogg'
+	miss_sound = 'sound/weapons/etherealmiss.ogg'
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/ethereal
+	mutantstomach = /obj/item/organ/stomach/ethereal
+	mutanttongue = /obj/item/organ/tongue/ethereal
+	exotic_blood = /datum/reagent/consumable/liquidelectricity //Liquid Electricity. fuck you think of something better gamer
+	siemens_coeff = 0.5 //They thrive on energy
+	brutemod = 1.25 //They're weak to punches
+	attack_type = BURN //burn bish
+	damage_overlay_type = "" //We are too cool for regular damage overlays
+	species_traits = list(MUTCOLORS, NO_UNDERWEAR, HAIR, HAS_FLESH, HAS_BONE) // i mean i guess they have blood so they can have wounds too
+	species_language_holder = /datum/language_holder/ethereal
+	inherent_traits = list(TRAIT_NOHUNGER)
+	sexes = FALSE
+	toxic_food = NONE
+	/*
+	citadel doesn't have per-species temperatures, yet
+	// Body temperature for ethereals is much higher then humans as they like hotter environments
+	bodytemp_normal = (BODYTEMP_NORMAL + 50)
+	bodytemp_heat_damage_limit = FIRE_MINIMUM_TEMPERATURE_TO_SPREAD // about 150C
+	// Cold temperatures hurt faster as it is harder to move with out the heat energy
+	bodytemp_cold_damage_limit = (T20C - 10) // about 10c
+	*/
```

Comment:
a good placeholder is to use what lizards and slimes have, which is a heat and cold damage modifier, also slimes have special lungs that take damage for breathing cold air earlier, too